pkg/api/dtos/index.go

```diff
@@ -14,6 +14,7 @@ type IndexViewData struct {
 	NewGrafanaVersionExists bool
 	NewGrafanaVersion       string
 	AppName                 string
+	IsSignedIn              bool
```

Comment:
This property is never assigned any value. 
You can use `User.IsSignedIn` instead.